modules/kn-trigger-list.adoc

```diff
@@ -8,24 +8,33 @@
 The `kn trigger list` command prints a list of available triggers.
 
 .Procedure
-
-* To print a list of available triggers, enter the following command:
+. Print a list of available triggers:
 +
+
+[source,terminal]
 ----
 $ kn trigger list
 ----
+
 +
-Example output:
+.Example output:
 +
+
+[source,terminal]
 ----
-$ kn trigger list
 NAME    BROKER    SINK           AGE   CONDITIONS   READY   REASON
 email   default   svc:edisplay   4s    5 OK / 5     True
 ping    default   svc:edisplay   32s   5 OK / 5     True
 ----
 
-* To print a list of triggers in JSON format, enter the following command:
+[id="kn-trigger-list-json_{context}"]
```

Comment:
one header/procedure per module: https://redhat-documentation.github.io/modular-docs/#module_anchor-and-file-names-concept